pkg/apis/certmanager/v1alpha1/types.go

```diff
@@ -139,6 +140,16 @@ type ACMEIssuerDNS01ProviderRoute53 struct {
 	Region          string            `json:"region"`
 }
 
+// ACMEIssuerDNS01ProviderRFC2136 is a structure containing the DNS
+// configuration for RFC2136
+type ACMEIssuerDNS01ProviderRFC2136 struct {
+	Nameserver    string            `json:"nameserver"`
+	TSIGSecret    SecretKeySelector `json:"TSIGSecretSecretRef"`
+	TSIGKey       string            `json:"TSIGKey"`
+	TSIGAlgorithm string            `json:"TSIGAlgorithm"`
+	Timeout       string            `json:"timeout"`
+}
```

Comment:
You'll need to run `./hack/update-codegen.sh` as you've updated API types.

Reply:
Hi munnerz,
I have been able to execute this command successfully.